lib/zipline/zip_generator.rb

```diff
@@ -15,13 +15,13 @@ def to_s
     def each(&block)
       fake_io_writer = ZipTricks::BlockWrite.new(&block)
       ZipTricks::Streamer.open(fake_io_writer) do |streamer|
-        @files.each {|file, name| handle_file(streamer, file, name) }
+        @files.each { |file, name, mtime| handle_file(streamer, file, name, modification_time: mtime) }
```

Comment:
i was expecting 
```suggestion
        @files.each { |file, name, options = {}| handle_file(streamer, file, name, options) }
```

Could you please write a test to verify that when you pass the expected

`modification_time: Time.parse("2008-06-21 13:30:00 UTC")` option that it passes it along correctly?
now i think it would pass

`modification_time: {modification_time: Time.parse("2008-06-21 13:30:00 UTC")}`

Time.parse("2008-06-21 13:30:00 UTC")

Reply:
Just committed some more changes and also added some tests.